openquakeplatform/openquakeplatform/templates/calculate.html

```diff
@@ -46,17 +46,10 @@
                 <li><a href="https://github.com/gem/oq-engine#installation" target="_blank">Download the OpenQuake Engine</a></li>
               </ul>
 
-            <b>Libraries:</b>
-              <ul>
-                <li>
-                  <a href="https://github.com/gem/oq-hazardlib" target="_blank">oq-hazardlib</a> - Openquake Hazard Library
-                </li>
-              </ul>
-
             <b>Modelling Tools:</b>
               <ul>
                 <li>
-                  <a href="https://github.com/GEMScienceTools/hmtk" target="_blank">Hazard Modeller's Toolkit</a> - A suite of tools used to create PSHA input models, mostly developed by GEM scientists
+                  <a href="http://docs.openquake.org/oq-hazardlib/stable/openquake.hmtk.html" target="_blank">Hazard Modeller's Toolkit</a> - A suite of tools used to create PSHA input models, mostly developed by GEM scientists
```

Comment:
Please add `It's provided by the <a href="https://github.com/gem/oq-engine#installation">OpenQuake Engine</a> packages.` too, as in my previous comment.